src/main/java/org/aesh/command/impl/Executions.java

```diff
@@ -232,7 +233,9 @@ public CommandResult getResult() {
                         if (config == null) {
                             throw new IllegalArgumentException("Invalid " + pl.line());
                         }
-                        config.setArgument(pl.line().trim());
+                        ParsedLineIterator iterator = pl.iterator();
+                        String word = iterator.hasNextWord() ? iterator.pollWord() : "";
+                        config.setArgument(word);
```

Comment:
pl.line() did contain unparsed content of the line. Therefore I used the first word from the ParsedLine and the rest is ignored (same behaviour can be observed in bash while running "echo version >> tmp\|file anotherWord".
Do you think this is good practise or it would be better to iterate over all the words and use all of them as an argument separated by white space?

Reply:
@jfdenise what do you think? The argument should only be one word afaik. It should be sufficient to call pl.lastWord() i think.

Reply:
I have looked at the bash operator, actually it can be located anywhere in the command.
For ex: > foo.txt echo bar or echo > foo.txt bar are identical.
aesh syntax doesn't support this and I think that we shouldn't this is not at all intuitive.
In this case we should get the next world and ignore the following words as it is suggested.

Reply:
@MMarus , you should log a bug against aesh, link the WFCORE-3554 issue to the aesh one and have the commit message to reference the aesh issue. It will be simpler to track aesh component upgrade.
Thanks.

Reply:
thanks @jfdenise for the insight. let's just use the first word then. @mmarus can you change the pr to just get the first word? then I think it looks ok.